### PR TITLE
add yarn team docs

### PR DESCRIPTION
**Pull request by nik-john**
### Overview

Added docs for yarn team. This section is currently missing documentation on the website.

## References

* [/src/cli/commands/team.js](https://github.com/yarnpkg/yarn/blob/master/src/cli/commands/team.js)

### CODE REVIEW

File: en/docs/cli/team.md [hunk 1]
```diff
@@ -8,8 +8,46 @@ layout: guide
 
 ##### `yarn team` <a class="toc" id="toc-yarn-team" href="#toc-yarn-team"></a>
 
-*Oops, something went wrong!*
+yarn team is a prefix used for a number of commands create destroy, add, rm and ls
```
Comment (thejameskyle) [inline en/docs/cli/team.md]: I don't think this is necessary to document, can just skip into the other commands

Comment (nik-john) [inline en/docs/cli/team.md]: Agreed. Will remove

File: en/docs/cli/team.md [hunk 2]
```diff
@@ -8,8 +8,46 @@ layout: guide
...
+
+##### `yarn team rm <scope:team> <user>` <a class="toc" id="toc-yarn-team-rm" href="#toc-yarn-team-rm"></a>
+
+Remove a user from a team they belong to.
+
+##### `yarn team ls <scope>|<scope:team>` <a class="toc" id="toc-yarn-team-ls" href="#toc-yarn-team-ls"></a>
+
+If performed on an organization name, will return a list of existing teams under that organization. If performed on a team, it will instead return a list of all users belonging to that particular team.
+
+### Details
```
Comment (nik-john) [inline en/docs/cli/team.md]: Ah my miss. Will update this!

**Comment by nik-john** [comment]
@thejameskyle You should see the review changes incorporated now. Once good , I'll rebase and squash

**Pull request merged**